imageProcessing/at-u2-v0-m.cpp

```diff
@@ -0,0 +1,670 @@
+            ("alpha,a", po::value<double>()->default_value( 1.0 ), "the parameter alpha (should be multiplied by N^2 where NxN is the size of the finest image)." )
+            ("epsilon,e", po::value<double>(), "the initial and final parameter epsilon of AT functional at the same time." )
+            ("epsilon-1", po::value<double>()->default_value( 2.0 ), "the initial parameter epsilon." )
+            ("epsilon-2", po::value<double>()->default_value( 0.25 ), "the final parameter epsilon." )
+            ("epsilon-r", po::value<double>()->default_value( 2.0 ), "sets the ratio between two consecutive epsilon values of AT functional." )
+            ("nbiter,n", po::value<int>()->default_value( 10 ), "the maximum number of iterations." )
+            ("image-snr", po::value<string>(), "the input image without deterioration if you wish to compute the SNR." )
+            ("pixel-size,p", po::value<int>()->default_value( 1 ), "the pixel size for outputing images (useful when one wants to see the discontinuities v on top of u)." )
+            ("color-v,c", po::value<string>()->default_value( "0xff0000" ), "the color chosen for displaying the singularities v (e.g. red is 0xff0000)." )
+            ("verbose,v", po::value<int>()->default_value( 0 ), "the verbose level (0: silent, 1: less silent, etc)." )
+            ("step,s", po::value<double>(), "the step size." )
+            ("multiresolution", po::value<string>(), "option for multiresolution." )
+            ("images-size,t", po::value<string>(), "sizes to run for multiresolution." )
+            ;
+
+    bool parseOK=true;
+    po::variables_map vm;
+    try {
+        po::store(po::parse_command_line(argc, argv, general_opt), vm);
+    } catch ( const exception& ex ) {
+        parseOK = false;
+        cerr << "Error checking program options: "<< ex.what()<< endl;
+    }
+    po::notify(vm);
+    if ( ! parseOK || vm.count("help") || !vm.count("input") )
+    {
+        cerr << "Usage: " << argv[0] << " -i toto.pgm\n"
+             << "Computes the Ambrosio-Tortorelli reconstruction/segmentation of an input image."
+             << "It outputs 2 or 3 images (of basename given by option --output) giving the"
+             << " reconstructed image u, and other images superposing u and the discontinuities v."
+             << endl << endl
+             << " / "
+             << endl
+             << " | a.(u-g)^2 + v^2 |grad u|^2 + le.|grad v|^2 + (l/4e).(1-v)^2 "
+             << endl
+             << " / "
+             << endl
+             << "Discretized as (u 2-form, v 0-form, A vertex-edge bdry, B edge-face bdy, M vertex-edge average)" << endl
+             << "E(u,v) = a(u-g)^t (u-g) +  u^t B diag(M v)^2 B^t u + l e v^t A^t A v + l/(4e) (1-v)^t (1-v)" << endl
+             << endl
+             << general_opt << "\n"
+             << "Example: ./at-u2-v0 -i ../Images/cerclesTriangle64b02.pgm -o tmp -a 0.05 -e 1 --lambda-1 0.1 --lambda-2 0.00001 -g"

```

Comment:
```suggestion
             << "Example: ./at-u2-v0-m -i ../Images/cerclesTriangle64b02.pgm -o tmp -a 0.05 -e 1 --lambda-1 0.1 --lambda-2 0.00001"
```
Since the -g is not used here..